### DISCUSSION TITLE
Users add content to a view and have the buttons in each view have their own values and can be used.

### DISCUSSION THREAD

**Discussion by lxj-py** [flet-dev/flet#831] [Q&A]
I wrote a music player and I added an ft. Row([ft. Card()]), and add icon buttons to it, I want to let users add content independently, and form a list in this form, how to make the value of the icon button in each item different after clicking, and switch the icon.

![~9W3P0$161A5PRU}X00POY](https://user-images.githubusercontent.com/79037707/211198071-ad4e4a20-792e-4992-bdd5-8b972b046b35.png)
![UZ %5MIAZNW~IJ_T%6S~JQ](https://user-images.githubusercontent.com/79037707/211198079-311f4317-5b1b-417e-a549-59ba94088867.png)

**Comment by FeodorFitsner** [comment]
Right, will be looking into that.

**Reply by lxj-py** [reply]
ok,thanks,That seems to be a difficult question.
